src/evergreen/api.py

```diff
@@ -58,12 +59,6 @@
 from evergreen.util import evergreen_input_to_output, format_evergreen_date, iterate_by_time_window
 from evergreen.version import RecentVersions, Requester, Version
 
-try:
-    from urlparse import urlparse
```

Comment:
I don't know what package this one was coming from but the urllib.parse package I started importing above definitely exists back to 3.9.